07-Modelling-Patchy-Particles/02-Simulating-a-System-of-Patchy-Particles.ipynb

```diff
@@ -0,0 +1,1909 @@
+{
```

Comment:
<p>I get that writing it this way better demonstrates the explanation in the above paragraph, but it feels awkward to not just write <code>sigma * kf_lambda</code> and in practice one would prefer to write the code in the more succinct way. It might be good to add a sentence at the end of the above paragraph just saying that sigma + sigma * (lambda - 1) = sigma * lambda and then write <code>r_cut = sigma * kf_lambda</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>